docs/configuration_file.md

```diff
@@ -57,6 +57,12 @@ Some options are also mandatory.
 
 * `client_cert`, `client_key` (*optional*, `string`) - paths to PEM-encoded client certificate and key to be used for authentication
 
+* `kerberos_keytab` (*optional*, `string`) - absolute path to a keytab that will be used to initialize kerberos credentials
+
+* `kerberos_principal` (*optional*, `string`) - kerberos principal for the keytab provided in `kerberos_keytab`
+
+* `kerberos_ccache` (*optional*, `string`) - absolute path to kerberos credential cache to use when `kerberos_keytab` is set (optional)
```

Comment:
Well, it's the kerberos location so it can be `KEYRING:...`. From what I tried if the `PREFIX:` is not given it is treated as `FILE:...`. Good point, I'll improve the docs.